docs/UserGuide.md

```diff
@@ -87,25 +87,33 @@ Format: `help`
 
 Adds a new worker to the McScheduler.
 
-Format: `worker-add n/NAME hp/PHONE_NUMBER a/ADDRESS [p/HOURLY_PAY] [r/ROLE]...`
+Format: `worker-add n/NAME hp/PHONE_NUMBER a/ADDRESS p/HOURLY_PAY [r/ROLE]... [u/UNAVAILABILITY]...`
 
-* Adds a worker with the specified `NAME`, `PHONE_NUMBER`, `ADDRESS` and `HOURLY_PAY`.
+* Adds a worker with the specified `NAME`, `PHONE_NUMBER`, `ADDRESS`, `HOURLY_PAY`, `ROLE`(s) and
+`UNAVAILABILITY`(s).
+* `HOURLY_PAY` can have a maximum of 2 decimal places only. 
 * The worker will be fit to take on the specified `ROLE`(s) in a shift. The specified `ROLE`(s) must be an existing role
   in the McScheduler. A role can be added to the McScheduler using the [role-add](#adding-a-role-role-add) command.
+* `UNAVAILABILITY` must be in the format {u/DAY TIME}. `DAY` should take one of these values: **Mon, Tue, Wed, Thu,
+Fri, Sat, Sun**. These values are case-insensitive (i.e. `Mon`, `MON`, `mon`, `mOn` etc. are all accepted). `TIME` should
+take one of these values: **AM, PM, FULL**. These values are also case-insensitive (i.e. `aM`, `fUll`, `pm`, etc. are all
```

Comment:
Assuming the suggestion above is taken, this can be split into 2 points for `UNAVAILABLE_DAY` and `UNAVAILABLE_TIME` which may be easier to read.